.github/workflows/release-winget.yaml

```diff
@@ -7,26 +7,30 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-    - name: Update winget repository
-      uses: mjcheetham/update-winget@v1.0
+    - id: update-winget
+      name: Update winget repository
+      uses: mjcheetham/update-winget@v1.2
       with:
-        token: ${{ secrets.WINGET_TOKEN }}
-        repo: microsoft/winget-pkgs
         id: Microsoft.GitCredentialManagerCore
+        token: ${{ secrets.WINGET_TOKEN }}   
         releaseAsset: gcmcore-win-x86-(.*)\.exe
+        repo: ldennington/winget-playground
         manifestText: |
-          Id: {{id}}
-          Version: {{version}}
-          Name: Git Credential Manager Core
+          PackageIdentifier: {{id}}
+          PackageVersion: {{version}}
+          PackageName: Git Credential Manager Core
           Publisher: Microsoft Corporation
-          AppMoniker: git-credential-manager-core
-          Homepage: https://aka.ms/gcmcore
-          Tags: "gcm, gcmcore, git, credential"
+          Moniker: git-credential-manager-core
+          PackageUrl: https://aka.ms/gcmcore
+          Tags: [ gcm, gcmcore, git, credential ]
           License: Copyright (C) Microsoft Corporation
-          Description: Secure, cross-platform Git credential storage with authentication to GitHub, Azure Repos, and other popular Git hosting services.
+          ShortDescription: Secure, cross-platform Git credential storage with authentication to GitHub, Azure Repos, and other popular Git hosting services.
```

Comment:
Is there now a longer `Description:` that we should populate somehow?

Reply:
[`Description` does still exist](https://github.com/microsoft/winget-cli/blob/cf6f8e58195bf033a1a5872f7b800866cd27a586/schemas/JSON/manifests/v1.0.0/manifest.singleton.1.0.0.json#L545-L555). However, for reasons that aren't super clear, `ShortDescription` is now required, while `Description` is not. I'm happy to take a stab at adding a longer `Description` though!

Reply:
Is there anything specific we'd want to call out in a longer `Description`?

Reply:
I think we can wait on `Description` until we know more about where it will show up.